src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php

```diff
@@ -141,4 +152,32 @@ protected function templateExists($template)
 
         return false;
     }
+
+    private function render(FlattenException $exception, Request $request): string
+    {
+        $statusCode = $exception->getStatusCode();
+        $logger = $request->attributes->get('logger');
+        $showException = $request->attributes->get('showException', $this->debug);
+        $currentContent = $this->getAndCleanOutputBuffering($request->headers->get('X-Php-Ob-Level', -1));
+
+        $template = $this->findTwigTemplate($statusCode, $request->getRequestFormat(), $showException);
+
+        if (null === $template) {
```

Comment:
The twig templates are deprecated which is good. But how would I be able to skip the deprecated templates and use the  ErrorRender instead? The templates will always be found because they are part of the twig bundle. IMO it should only use the twig templates if people overwrote them in their apps. That is why a new controller as suggested would make this possible using a new template location, e.g. `@Twig/Error/` instead of `@Twig/Exception/`.

Reply:
That's a better migration path, I will do the changes, thank you!